<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModBusClient</class>
 <widget class="QWidget" name="ModBusClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Параметры соединения</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="connect_layout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Редактирование тегов</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
         <item>
          <widget class="TreeView" name="treeView">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QTabWidget" name="tabWidget_2">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="addTag_tab">
              <attribute name="title">
               <string>Добавление</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <widget class="QGroupBox" name="groupBox">
                 <property name="title">
                  <string>Добавить тэг</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <layout class="QGridLayout" name="gridLayout_2">
                    <item row="3" column="1">
                     <widget class="QComboBox" name="type_cmbox">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="id_lbl">
                      <property name="text">
                       <string>Идентификатор</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLineEdit" name="id_lnedit"/>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="name_lbl">
                      <property name="text">
                       <string>Наименование</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLineEdit" name="name_lnedit"/>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="type_lbl">
                      <property name="text">
                       <string>Тип</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0">
                     <widget class="QLabel" name="mask_lbl">
                      <property name="text">
                       <string>Маска</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1">
                     <widget class="QLineEdit" name="mask_lnedit"/>
                    </item>
                    <item row="5" column="0">
                     <widget class="QLabel" name="description_lbl">
                      <property name="text">
                       <string>Описание</string>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="1">
                     <widget class="QLineEdit" name="description_lnedit"/>
                    </item>
                    <item row="7" column="1">
                     <widget class="QPushButton" name="addTagBefore_btn">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>143</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Добавить перед текущим</string>
                      </property>
                     </widget>
                    </item>
                    <item row="8" column="1">
                     <widget class="QPushButton" name="addTagAfter_btn">
                      <property name="text">
                       <string>Добавить после текущего</string>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="1">
                     <widget class="QCheckBox" name="shift_ckbox">
                      <property name="text">
                       <string/>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="0">
                     <widget class="QLabel" name="shift_lbl">
                      <property name="text">
                       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Объединить&lt;/p&gt;&lt;p&gt;с текущим&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="LineEdit" name="bind_lnedit">
                      <property name="clearButtonEnabled">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QLabel" name="bind_lbl">
                      <property name="text">
                       <string>Привязка</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="controlTag_tab">
              <attribute name="title">
               <string>Управление</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_16">
               <item>
                <widget class="QGroupBox" name="groupBox_4">
                 <property name="title">
                  <string>Управление тегами</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QPushButton" name="up_btn">
                    <property name="text">
                     <string>Вверх</string>
                    </property>
                    <property name="icon">
                     <iconset>
                      <selectedon>:/icons/move_up.png</selectedon>
                     </iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="down_btn">
                    <property name="text">
                     <string>Вниз  </string>
                    </property>
                    <property name="icon">
                     <iconset>
                      <selectedon>:/icons/move_down.png</selectedon>
                     </iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="deleteTag_btn">
                    <property name="text">
                     <string>Удалить</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="fileHandle_tab">
              <attribute name="title">
               <string>Файл</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <item>
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="title">
                  <string>Файл описания базы modbus</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QPushButton" name="openFile_btn">
                    <property name="text">
                     <string>Импортировать файл</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="saveFile_btn">
                    <property name="text">
                     <string>Экспортировать файл</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="operationsTag_tab">
              <attribute name="title">
               <string>Коррекция</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_18">
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string>Изменение идентификатора</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_14">
                  <item>
                   <widget class="QRadioButton" name="id_choose_suffix_rb">
                    <property name="text">
                     <string>суффикс</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="id_choose_prefix_rb">
                    <property name="text">
                     <string>префикс</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="suffix_or_preffix_lnedit"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="set_suffix_or_prefix_btn">
                    <property name="text">
                     <string>Установить</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="delete_suffix_or_prefix_btn">
                    <property name="text">
                     <string>Удалить</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>src/widgets/tree/view/treeview.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/widgets/lineedit/lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
